### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by chaitanyanarukulla**

### CODE REVIEW

File: src/main/resources/Static/style.css [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  opacity: 0;
+  position: relative;
+  width: 64px;
+}
+input[type="checkbox"]:checked {
+  left: 29px;
+  width: 58px;
+}
+.toggle {
+  background: url(http://i.minus.com/ibitS19pe8PVX6.png) no-repeat;
```
Comment (stephenchu530) [inline src/main/resources/Static/style.css]: What is this? The link is busted?
Image Link

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,2 +1,7 @@
-spring.datasource.url=${DATABASE_URL}
+#spring.datasource.url=${DATABASE_URL}
+#spring.jpa.hibernate.ddl-auto=update
+
+
+spring.datasource.url=jdbc:postgresql://localhost:5432/pawpal
```
Comment (stephenchu530) [inline src/main/resources/application.properties]: Don't include this. This is wrong

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -1,2 +1,7 @@
...
+
 spring.jpa.hibernate.ddl-auto=update
```
Comment (stephenchu530) [inline src/main/resources/application.properties]: Redundant

File: src/main/resources/Static/style.css (hunk 1 above)
Comment (chaitanyanarukulla) [reply src/main/resources/Static/style.css]: will update with pics

**Pull request merged**